RELEASE_PROCEDURE.md

```diff
@@ -0,0 +1,36 @@
+# Release procedure for DLA-Future
+
+DLA-Future follows [Semantic Versioning](https://semver.org).
+
+1. For minor and major releases: create and check out a new branch at an
+   appropriate point on `main` with the name `release-major.minor`.  `major` and `minor` should be the
+   major and minor versions of the release. For patch releases: check out the corresponding
+   `release-major.minor` branch.
+
+1. Write release notes in `CHANGELOG.md`. Check for issues and pull requests for the release on the
+   [DLA-F Planning board](https://github.com/orgs/eth-cscs/projects/1). All list entries and paragraphs
```

Comment:
```suggestion
   [DLA-Future Planning board](https://github.com/eth-cscs/DLA-Future/projects/2). All list entries and paragraphs
```

Reply:
The on you link to is the "old" project board. Perhaps we can close or delete it now? @rasolca do you mind if we close it?

`DLA-F Planning board` matches the title of the new project board, but maybe we just update the title there to be `DLA-Future Planning board`. Seems more consistent?

Reply:
I see. The other is a dangling link though. Maybe I don't have the right permissions? Happy to leave `DLA-F` if that's the actual name of the board (which I was not able to access).

Reply:
Aha, yes, I just checked and it was private, not including you! Sorry about that... Can you see it now? I made it public, since it just contains all the issues/PRs that we anyway have publicly visible.